shell/tools/build-strategy-explorer

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+cd components/strategy-explorer
```

Comment:
I think this will leave us in the `components/strategy-explorer` directory when finished. Maybe won't if we wrap it with parentheses like `(cd components/strategy-explorer && bower install)`. Does it matter? I am not sure.

Reply:
It doesn't - the script is executed in a subshell.

But, it was a good question because when I checked to make sure I found that the script doesn't work outside of `npm`:

```
shell> tools/build-strategy-explorer 
tools/build-strategy-explorer: line 4: bower: command not found
shell> 
```

I'm not sure that matters but it's an easy fix.

Reply:
(at least on Linux - it's more complex on Windows so I'm not going to dig in there).